lint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: "./lint.sh" (equivalent to "./lint.sh check"), "./lint.sh fix"
+# To download the dependencies as well: "DOWNLOAD=true ./lint.sh ...".
+
+set -eou pipefail
+
+# Where to put google-java-format
+DESTDIR=$HOME
+
+if "${DOWNLOAD:-false}"; then
+  go get -u github.com/bazelbuild/buildifier/buildifier
+  curl -L https://github.com/google/google-java-format/releases/download/google-java-format-1.5/google-java-format-1.5-all-deps.jar > $DESTDIR/google-java-format-1.5-all-deps.jar
```

Comment:
add a go get for buildifier too?

Reply:
oh sorry, please disregard that comment, the first line IS downloading buildifier, that's what I was saying :) I misread it :sweat_smile: